content/en/blog/_posts/2023-08-25-sidecar-kep.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Introducing native sidecar containers"
+date: 2023-08-25
```

Comment:
we need to fix this.

Reply:
Why is this date wrong?

Reply:
> Why is this date wrong?

Couldn't this blog be released today?

Reply:
> Couldn't this blog be released today?

Sure, but you'll need to ask release comms for an exception (and then get that exception).

Reply:
> https://github.com/orgs/kubernetes/projects/140/views/6 lists the 25th of August.

Thanks, I didn't realize that we have the scheduled date :)